tests/MongoDBDriver_APM-001.phpt

```diff
@@ -0,0 +1,578 @@
+--TEST--
+HHVM-190: APM Specification
```

Comment:
This is only the first test. I have no tests for any of the accessors either. I would like to keep this one test, but add more for each of the other specific parts - and then also add them for each accessor method.